File: EditImageView/src/main/java/com/image/edit/helper/BitmapHelper.java
package com.image.edit.helper;

import android.graphics.Bitmap;
import android.support.annotation.Nullable;

/**
 * @author y
 * @create 2018/11/20
 */
public class BitmapHelper {

    public static void recycle(@Nullable Bitmap bitmap) {
        if (bitmap != null && !bitmap.isRecycled()) {
            bitmap.recycle();
        }
    }

}

File: EditImageView/src/main/java/com/image/edit/helper/BitmapHelper.kt
package com.image.edit.helper

import android.graphics.Bitmap

/**
 * @author y
 * @create 2018/11/20
 */
object BitmapHelper {

    fun recycle(bitmap: Bitmap?) {
        if (bitmap != null && !bitmap.isRecycled) {
            bitmap.recycle()
        }
    }

}
